src/crypto.ts

```diff
@@ -71,3 +75,50 @@ export async function loadRifRollupCrypto(wasmFileUrl?: string) {
         rifRollupCryptoLoaded = true;
     }
 }
+
+const validateWasmFileUrl = (wasmFileUrl: string) => {
+    // Regular expression pattern to match a valid URL format
+    const urlPattern = new RegExp(
```

Comment:
If we're validating a regular URL only, I think it would be better to use the [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) constructor. Regexps are quite difficult to maintain, and they may lead to security issue also in case the URL is received from a URL; @0xM3R WDYT?

Reply:
Fine! That should cover the input space